src/main/kotlin/no/nav/sifinnsynapi/sak/SakService.kt

```diff
@@ -65,7 +69,7 @@ class SakService(
 
```

Comment:
Kommentar til linje 64
``` val ikkeSkjermetOmsorgsbarn = oppslagsbarn
            // Dersom pleietrengende er skjermet, vil ikke hen returneres fra oppslagstjenesten.
            // Vi sjekker derfor opp mot pleietrengende søker har omsorgen for, og returnerer kun de som ikke er skjermet.
            .filter { pleietrengendeSøkerHarOmsorgFor.contains(it.aktørId) }
        logger.info("Fant ${ikkeSkjermetOmsorgsbarn.size} pleietrengende som vi kan hente saker for.")
```

Hvis borger har pleietrengende (f.eks. punsja manuelt), men ingen folkeregistrerte barn så vil `ikkeSkjermetOmsorgsbarn` returne tom liste. Går det an å helle gjøre oppslag mot pdl på pleietrengende sin aktørid for å sjekke skjermingsstatus?